test/evergreen.yml

```diff
@@ -2608,6 +2608,23 @@ buildvariants:
     - name: static-wt-build-test
     - name: format-failure-configs-test
 
+- name: ubuntu1804-cmake
+  display_name: "! Ubuntu 18.04 CMake"
+  run_on:
+  - ubuntu1804-test
+  expansions:
+    test_env_vars: LD_LIBRARY_PATH=$(pwd)
+    posix_configure_flags: -DHAVE_DIAGNOSTIC=1 -DENABLE_PYTHON=1 -DENABLE_ZLIB=1  -DENABLE_SNAPPY=1 -DENABLE_STRICT=1 -DCMAKE_INSTALL_PREFIX=$(pwd)/LOCAL_INSTALL
+    python_binary: '/opt/mongodbtoolchain/v3/bin/python3'
+    smp_command: -j $(echo "`grep -c ^processor /proc/cpuinfo` * 2" | bc)
+    cmake_generator: Ninja
+    make_command: ninja
+    is_cmake_build: true
+  tasks:
+    - name: compile
+    - name: make-check-test
+    - name: ".unit_test"
```

Comment:
`.unit_test` tag selector would be mapped to the 11 `unit-test-bucketxx` tasks. The reason we broke `unit-test` into those buckets was to have a higher level of parallelism so that each task can run in parallel to save overall "make span", which is specifically used for pull request testing. As I don't think we plan to add any cmake task into pull request testing (yet), putting a `unit-test` task (sum of the 11 bucket tasks) should be good enough.

Reply:
Thanks for the context, I initially used `.unit_test` for the speed up benefit but didn't consider the added resource cost. I've updated the cmake variants to use `unit-test`.